### PR TITLE
add issue_template.md

### PR DESCRIPTION
**Pull request by rborn**

### CODE REVIEW

**Comment by rborn** [comment]
@alvelig 😻

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+<!--
+  Please specify the react, react-native, react-native-maps versions, eg:
+
+    react-native: 0.51.0
+    react: 16.0.0
+	react-native-maps: 0.18.3 or "git+ssh://git@github.com/react-community/react-native-maps.git"
+
+  ***Make sure you are on v0.48.0 or greater of react-native, otherwise you may get this error:
+
+  Target Platform: eg: iOS - 11.2
```
Comment (alvelig) [inline .github/ISSUE_TEMPLATE.md]: Simulator / iPhone 6s

**Comment by alvelig** [comment]
Great.

**Comment by rborn** [comment]
@alvelig btw, who can publish to npm ? we'll need to push some new version soon :)

File: .github/ISSUE_TEMPLATE.md (hunk 1 above)
Comment (rborn) [reply .github/ISSUE_TEMPLATE.md]: done

**Comment by christopherdro** [comment]
I’ll be able to do a release just ping me when ready.

For then issue template I think we should make sure to ask if user is using  google maps or native on iOS.

**Comment by rborn** [comment]
@christopherdro @alvelig done, anything else? Thnx 🤗

**Pull request merged**